enterprise/cmd/cody-gateway/shared/config.go

```diff
@@ -85,6 +88,9 @@ func (c *Config) Load() {
 
 	c.Trace.Policy = policy.TracePolicy(c.Get("CODY_GATEWAY_TRACE_POLICY", "all", "Trace policy, one of 'all', 'selective', 'none'."))
 	c.Trace.GCPProjectID = c.Get("CODY_GATEWAY_TRACE_GCP_PROJECT_ID", os.Getenv("GOOGLE_CLOUD_PROJECT"), "Google Cloud Traces project ID.")
+
+	c.ConcurrentLimit.Percentage = float32(c.GetPercent("CODY_GATEWAY_CONCURRENT_LIMIT_PERCENTAGE", "50", "The percentage of daily rate limit to be allowed as concurrent requests limit.")) / 100
+	c.ConcurrentLimit.Interval = c.GetInterval("CODY_GATEWAY_CONCURRENT_LIMIT_INTERVAL", "10s", "The interval at which to check the concurrent requests limit.")
```

Comment:
nit - clarify that this concurrency is per-actor:

```suggestion
	c.ActorConcurrentLimit.Percentage = float32(c.GetPercent("CODY_GATEWAY_ACTOR_CONCURRENT_LIMIT_PERCENTAGE", "50", "The percentage of daily rate limit to be allowed as concurrent requests limit from an actor.")) / 100
	c.ConcurrentLimit.Interval = c.GetInterval("CODY_GATEWAY_ACTOR_CONCURRENT_LIMIT_INTERVAL", "10s", "The interval at which to check the concurrent requests limit from an actor.")
```